src/main/webapp/app/shared/notification/notification.service.ts

```diff
@@ -62,7 +62,12 @@ export class NotificationService {
         if (notification.target) {
             const target = JSON.parse(notification.target);
             const courseId = target.course || notification.course?.id;
-            this.router.navigate([target.mainPage, courseId, target.entity, target.id]);
+
+            if ((notification.title = 'Quiz Started')) {
```

Comment:
minor thing: double round brackets were forgotten to be removed

Reply:
Is this assignment of `notification.title` on purpose?